src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+abstract class Figure implements Drawable {
```

Comment:
Stick to the task description
we can obtain the area of any figure and are able to draw it. 

Add area to the draw method for each type of figure